engines/python/setup/djl_python/huggingface.py

```diff
@@ -593,9 +497,14 @@ def handle(inputs: Input):
     """
     Default handler function
     """
+    global _service
     if not _service.initialized:
+        properties = inputs.get_properties()
+        if "rolling_batch" in properties and properties.get(
+                "rolling_batch") != RollingBatchEnum.disable.value:
+            _service = RollingBatchService()
         # stateful model
-        _service.initialize(inputs.get_properties())
```

Comment:
Basically, you made the RollingBatch class a top level entryPoint
Let's have a design discussion about this decision.

Reply:
Sure. Not really a top level entryPoint. RollinBatch does not have `handle` method in it. And the UX still does not change, entryPoint would still be `djl_python.huggingface` for all the vllm, lmi-dist, scheduler rolling batch entryPoint.